composer.json

```diff
@@ -25,7 +26,9 @@
         "pusher/pusher-php-server": "^2.3",
         "zizaco/entrust": "1.7.*",
         "laravel/tinker": "^1.0",
-        "laravel/dusk": "^1.0"
+        "laravel/dusk": "^1.0",
+        "doctrine/dbal": "^2.9",
```

Comment:
Why are we adding doctrine? Where is it even used?

Reply:
Doctrine DBAL allows you to modify existing database columns using a migration.

See here: https://laravel.com/docs/5.4/migrations#modifying-columns

I used DBAL's change() method in 2019_05_05_233153_change_contact_field_name.php.